src/org/opensolaris/opengrok/util/Executor.java

```diff
@@ -136,6 +136,21 @@ public int exec(boolean reportExceptions) {
         return ret;
     }
 
+    /**
+     * Close all the 3 streams of a process.
+     * @param process
+     */
+    private static void closeStreams(Process process) {
+        try {
+            process.getOutputStream().close();
```

Comment:
Is this really necessary? `Process` API was refactored in Java 9 and I have basically the same code in Java 10 as is here. As you can see, there is different handling for `Solaris`, it might be a good to investigate why. Maybe they have good reason.

<img width="1440" alt="screen shot 2018-03-29 at 14 03 44" src="https://user-images.githubusercontent.com/12401160/38087954-69e0cee2-335a-11e8-904c-77cfa30799b4.png">

Reply:
In Java 8 Netbeans complain if the try/catch block for the IOException is not there.

Reply:
No, I mean what is the purpose of closing the streams right before the `destroy()` call which under the hood does exactly the same. Does `destroy()` throw an exception which prevents closing the streams?